client/v3/op.go

```diff
@@ -391,6 +391,7 @@ func WithPrefix() OpOption {
 	return func(op *Op) {
 		if len(op.key) == 0 {
 			op.key, op.end = []byte{0}, []byte{0}
+			op.isOptsWithPrefix = true
```

Comment:
minor comment: Both line 394 and 397 set `isOptsWithPrefix` as `true`, so I suggest to move 			`op.isOptsWithPrefix = true` to the beginning of the function.

Reply:
@ahrtr Good catch, I missed it. Thanks!